SCClassLibrary/Common/Streams/Rest.sc

```diff
@@ -1,56 +1,23 @@
-Rest {
-	var <>dur = 1;
-	*new { |dur(1)|
-		^super.newCopyArgs(dur)
-	}
-	// for use by Pfunc and Pfuncn
-	*processRest { |inval|
-		inval.put(\isRest, true);
-		^1
-	}
-	processRest { |inval|
-		inval.put(\isRest, true);
-		^dur.value(inval)
-	}
-	// for use e.g. in ListPatterns
-	*embedInStream { |inval|
-		^this.processRest(inval).yield;  // the actual return value is the next inval
-	}
-	embedInStream { |inval|
-		^this.processRest(inval).yield;
-	}
-	*asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
-	}
-	asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
+
+Rest : Operand  {
+
+	*new { |value = 1|
+		^super.newCopyArgs(value.dereferenceOperand).unwrapBoolean
```

Comment:
I don't disagree that it's dodgy to newCopyArgs here. Cleaning up questionable uses of it is a bigger project though... I'd suggest, for this PR, leave it as is and open another issue for review of newCopyArgs vs inheritance.